apps/studio/pages/project/[ref]/api/index.tsx

```diff
@@ -66,14 +66,9 @@ const DocView = observer(() => {
   const [selectedLang, setSelectedLang] = useState<any>('js')
   const [showApiKey, setShowApiKey] = useState<any>(DEFAULT_KEY)
 
-  const { data, error } = useProjectApiQuery({
-    projectRef,
-  })
-
+  const { data, error } = useProjectApiQuery({ projectRef })
   const apiService = data?.autoApiService
-  const anonKey = apiService?.service_api_keys.find((x) => x.name === 'anon key')
-    ? apiService.defaultApiKey
-    : undefined
+  const anonKey = apiService?.defaultApiKey ?? undefined
```

Comment:
this is the main fix here, the rest are just rearranging stuff to make sure return statements are all the way at the bottom

Reply:
was thinking here that the .find seems really unnecessary - but please lemme know if im overlooking something